common/src/main/java/com/box/l10n/mojito/okapi/extractor/AssetExtractionStep.java

```diff
@@ -80,6 +82,22 @@ protected Event handleTextUnit(Event event) {
     return eventToReturn;
   }
 
+  @Override
+  protected Event handleDocumentPart(Event event) {
```

Comment:
(Added some docs in the code...)

that's the code use to extract the image src attribute. That attribute is not processed as text unit since it is not truly something to be translated. Instead it is in the document part, but can be customized. 

There are many document part, only a very few will have something to be localized. The filter set the annotation to indicate a property to localized. If documentPartPropertyAnnotation is null then i means there is nothing to localize in the document part

Reply:
Not sure if the documentation was pushed yet - flagging just in case as I can't see it for this function.

Reply:
I added the documentation in the parent class: `AbstractMd5ComputationStep`